### PR TITLE
Move "game logic" to for loop checking user input

### PR DESCRIPTION
**Pull request by gregcorbett**
The example on page 329 of the book uses the layout below for the game loop, so let's follow it.

```
while true:
    for each event:
        check user input
    update game state and screen
```

### CODE REVIEW

File: PythonLessons/PyGame/pygame-lesson1.py [hunk 1]
```diff
@@ -19,28 +19,29 @@
...
 
+while True:
     # Check for user input
     for event in pygame.event.get():
         # Check to see if the "Close" button has been pressed
         if event.type == pygame.QUIT:
             pygame.quit()
             sys.exit(0)
+
+        # Fill the screen to prevent any moving shapes leaving a 'trail'
```
Comment (tofu-rocketry) [inline PythonLessons/PyGame/pygame-lesson1.py]: This section below is now within the `for...loop`. Was that intended? It doesn't seem to use any `event`s.

Comment (norealroots) [reply PythonLessons/PyGame/pygame-lesson1.py]: Using the example Greg said: 
```
while true:
   for each event:
        check user input
    update game state and screen
```

I agree, It doesn't seem like it should be inside the `for...loop`.

**Review by norealroots — CHANGES_REQUESTED** [review]
It all looks good, but if you could move the
`update game state and screen`
section out from within the for loop first that'd be great!

File: PythonLessons/PyGame/pygame-lesson1.py (hunk 1 above)
Comment (gregcorbett) [reply PythonLessons/PyGame/pygame-lesson1.py]: Yes, the original intention was to move it below the `for...loop` not within it. Changed.

**Review by norealroots — APPROVED** [review]

**Pull request merged**